setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
Maybe change docstring from: `You can override this fixture to return base of your model` to `You need to override this fixture to return declarative base of your model` ...